src/view/components/rule-engine/condition-types/serverside-condition/serverside-condition.ts

```diff
@@ -86,10 +102,24 @@ export class ServersideCondition {
     this._inputs = [];
   }
 
+  setVisible(value, input):void {
+    if(!value) return
+    if(input.name === 'comparison'){
+      let idx = this._inputs.indexOf(input)
+      let comparisonObj = input.options.filter((e)=> { return e.value == value })[0]
```

Comment:
hah. who knows. Maybe I just realized how awesome it is to not have distracting 'return' keywords in the middle of a large code block. ;~)